.pre-commit-config.yaml

```diff
@@ -2,16 +2,14 @@
 #   pre-commit install
 # To update the pre-commit hooks run:
 #   pre-commit install-hooks
-exclude: '^(\.tox|ci/templates|\.bumpversion\.cfg)(/|$)'
+exclude: '^(\.tox|ci/templates|\.bumpversion\.cfg|\.patch)(/|$)'
```

Comment:
Excluded `.patch` here. Not sure if we need to exclude this anywhere else

Reply:
This excludes a single file at the root named `.patch`, I believe. I think you need something like:
```suggestion
exclude: '^(\.tox|ci/templates|\.bumpversion\.cfg|.*\.patch)(/|$)'
```

Reply:
Thanks Bradley! 🙏

Committed this change